starburst-presto/presto.sh

```diff
@@ -36,6 +36,26 @@ function err() {
   return 1
 }
 
+function retry_apt_command() {
+  cmd="$1"
+  for ((i = 0; i < 10; i++)); do
+    if eval "$cmd"; then
+      return 0
+    fi
+    sleep 5
+  done
+  return 1
+}
+
+function update_apt_get() {
+  retry_apt_command "apt-get update"
+}
+
+function install_apt_get() {
+  local pkgs="$*"
+  retry_apt_command "apt-get install -y $pkgs"
+}
```

Comment:
once you remove java 11 installation, you can also remove this

Reply:
Done.